fe/fe-core/src/main/java/com/starrocks/catalog/DataProperty.java

```diff
@@ -98,20 +98,35 @@ public static DataProperty getInferredDefaultDataProperty() {
 
         Preconditions.checkState(mediumSet.size() <= 2, "current medium set: " + mediumSet);
         TStorageMedium m = TStorageMedium.SSD;
+        long cooldownTimeMs = MAX_COOLDOWN_TIME_MS;
         // When storage_medium property is not explicitly specified on creating table, we infer the storage medium type
         // based on the types of storage paths reported by backends. Here is the rules,
         //   1. If the storage paths reported by all the backends all have storage medium type HDD,
         //      we infer that user wants to create a table or partition with storage_medium=HDD.
         //   2. If the reported storage paths have both SSD type and HDD type, and storage cool down feature is
         //      not used, we also infer with HDD type.
         //   3. In other cases, it's SSD type.
+
         if (mediumSet.size() == 0 ||
                 (mediumSet.size() == 1 && mediumSet.iterator().next() == TStorageMedium.HDD) ||
                 (mediumSet.size() == 2 && Config.tablet_sched_storage_cooldown_second < 0)) {
             m = TStorageMedium.HDD;
         }
 
-        return new DataProperty(m);
+        if (mediumSet.size() == 2) {
```

Comment:
```suggestion
        if (m == TStorageMedium.SSD) {
```

only `SSD` medium type need to set cool down time based on `Config.tablet_sched_storage_cooldown_second`

Reply:
i see